completions/asdf.completion.sh

```diff
@@ -0,0 +1,8 @@
+#! bash oh-my-bash.module
+# Bash completion support for the `asdf` command.
+# Depends on the `asdf` plugin.
+
+# Only load the completions if the ASDF_DIR variable was set.
+if [[ -v ASDF_DIR ]]; then
```

Comment:
Oh My Bash targets Bash >= 3.2 while `[[ -v ASDF_DIR ]]` is only available in Bash >= 4.2.

```suggestion
if [[ ${ASDF_DIR+set} ]]; then
```